api/controllers/SdtdServer/countryBan/disable-country-whitelist.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+
+  friendlyName: 'Disable country whitelist',
+
+  description: 'Set the country banlist to act as a blacklist',
+
+  inputs: {
+    serverId: {
+      type: 'number',
+      required: true
+    }
+  },
+
+  exits: {
+    success: {
+    },
+  },
+
+
+  fn: async function (inputs, exits) {
+    let config = await SdtdConfig.findOne({ server: inputs.serverId });
```

Comment:
`config` might be null/undefined. Probably best to add a check for that so the controller can exit 400 instead of error out. Same for the enable controller

Reply:
How should that be done? I can't seem to find any example of an exit 400

Reply:
400 = badRequest

https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/blob/b7b049c6e93898fe3daf5f9e0ed49e5b5f852668/api/controllers/SdtdServer/dashboard.js#L47

Make sure to register that exit at the top as well